lib/pbench/server/api/resources/elasticsearch_api.py

```diff
@@ -0,0 +1,81 @@
+import requests
+from flask_restful import Resource, abort
+from flask import request, make_response
+from flask_cors import cross_origin
+
+
+class Elasticsearch(Resource):
+    """Elasticsearch API for post request via server."""
+
+    def __init__(self, elasticsearch_uri, logger, api):
+        self.elasticsearch = elasticsearch_uri
```

Comment:
This looks like a specific configuration that the `Elasticsearch` class could pull out of the configuration object so that knowledge of this class remains with the class.

Reply:
yeah I agree, I did that before where I was passing whole config object and elasticsearch class will fetch the required parameters but then I changed it to something like give me only what I want framework, but you put a good point that this is a specific configuration for the Elasticsearch class and let it fetch what it wants